schedule/yast/sle/flows/default_s390x_kvm.yaml

```diff
@@ -48,6 +48,7 @@ confirm_reboot:
   - installation/performing_installation/confirm_reboot
 reconnect_svirt:
   - installation/performing_installation/reconnect_after_reboot
+encryption: []
```

Comment:
encryption is too generic, there could be encryption related steps in many places, let's use `disk_passphrase`

Reply:
Changed and please take a look again,  Thanks!